python/benchmarks/basic_functions.py

```diff
@@ -15,12 +16,12 @@ class BasicFunctions:
     number = 5
     timeout = 6000
 
-    params = ([100_000, 150_000], [500, 1000])
+    params = ([10_000, 15_000], [500, 1000])
```

Comment:
Why this change? IMO these row counts are already very low, and so not testing very representative use cases

Reply:
I just wanted to see what would be the time difference with the smaller rows.
The benchmarks runs around 20-25 min faster with the smaller values, which isn't that much faster (~140 vs ~160 min total run time).

I have brought the old values back.